package.json

```diff
@@ -21,17 +21,19 @@
   "dependencies": {
     "enhanced-resolve": "^3.1.0",
     "es6-promisify": "^5.0.0",
-    "icss-utils": "^4.0.0",
+    "icss-utils": "5.1.0",
```

Comment:
Should probably use caret `^` for consistency, unless there's a reason not to.

Reply:
Done ✅